etl/etl.py

```diff
@@ -14,15 +14,49 @@ def extract_from_nix():
     )
     print("Downloading data from nix as a dataframe...")    
     result = subprocess.run('NIXPKGS_FLAKE_REF="github:nixos/nixpkgs/master" nix eval --json --file '+ str(nixpkgs_graph_nix_file_path) + ' --extra-experimental-features "nix-command flakes"', shell=True, stdout=subprocess.PIPE)
-    dataframe = pd.read_json(result.stdout.decode(), orient="records")    
+    dataframe = pd.read_json(result.stdout.decode(), orient="records")
     return dataframe
 
 def data_process(dataframe):
-    dataframe["path"] = dataframe.groupby(["outputPath"])["path"].transform(lambda x : ", ".join(x))
+    dataframe["path"] = dataframe.groupby(["outputPath"])["path"] \
+                                 .transform(lambda x : ", ".join(x))
```

Comment:
Only one formatter should be used to avoid this kind of diff

Reply:
Good point! :+1: